lms/static/sass/multicourse/_entitlement_dashboard.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper-entitlement-action-more {
+    @extend .wrapper-action-more;
+
+    .entitlement-action {
+        @extend .action;
+    }
+
+    .entitlement-action-more {
+        @extend .action-more;
+    }
+}
+
+.entitlement-actions-dropdown {
+    @extend .actions-dropdown;
+
+    .entitlement-actions-dropdown-list {
+        @extend .actions-dropdown-list;
+
+        .entitlement-actions-item {
+            @extend .actions-item;
+
+            .entitlement-action {
+                @extend .action;
+            }
+        }
+    }
+}
+
+.entitlement-modal {
+    @extend .modal;
+
+    .entitlement-modal-inner-wrapper {
+        @extend .inner-wrapper;
+
+        .close-entitlement-modal {
+            @extend .close-modal;
+        }
+
+        .entitlement-unenroll-header {
+            @extend .unenroll-header;
+        }
+
+        .entitlement-modal-form-error {
+            @extend .modal-form-error;
+        }
+    }
+}
```

Comment:
Note: the goal here is to achieve the same styles as the original modal/gear dropdown without re-using any of the existing class names. There is a lot of JS that runs on the dashboard and hooks onto elements with those classnames for things like analytics, which we are not ready to support for entitlements. 

This seems to work in that the modal/dropdown look correct when these styles are applied like this. I am not very good with sass though, so if there are issues with this approach, please let me know.

@HarryRein , @AlasdairSwan

Reply:
Can you give an example of somewhere this issue arises (with linking js objects) ? It seems like there may be a way to handle the issues on the lack of specificity in the JS without having to add this file.

Reply:
https://github.com/edx/edx-platform/blob/master/lms/static/js/dashboard/track_events.js#L91 hooks onto `.wrapper-action-more` to add analytics tracking.

Reply:
Gotcha. If you think you can find all the instances that these issues occur (only here?) then you can add a parent selector or something to make it more distinct.

Reply:
@HarryRein I'm not sure how to definitively find all the instances where we have things hooking into these elements, and with the deadline fast approaching it would be difficult to exhaustively search through the codebase for each of these classes. That's why I took the approach of just namespacing them for now and replicating the behavior/styles.